client/src/components/Proverbs/Addproverbs.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import {
+  Modal, Button, Input, message,
+} from 'antd';
+import './proverb.css';
+import http from '../../services/httpService';
+
+function Addproverbs() {
+  const [visible, setVisible] = useState(false);
+  const [value, setValue] = useState({ proverb: '' });
+  const handleChange = (e) => setValue({ proverb: e.target.value });
+  console.log(value);
+
+  const sendProverb = async () => {
+    try {
+      if (value.proverb !== '') {
+        const proverb = await http.post('api/v1/proverb', value);
+        message.success(proverb.response.data.message);
+      } else {
+        setVisible(true);
+        throw new Error('يجب عليك إضافة مثل');
```

Comment:
why to throw instead use messsgae.warn here,
OR I prefer to validate input using Form.Item